incubating/argo-cd-sync/argocd_sync.py

```diff
@@ -0,0 +1,93 @@
+from gql import Client, gql
+from gql.transport.requests import RequestsHTTPTransport
+import os
+
```

Comment:
Should we add some kind of logging code?
and a LOG_LEVEL option?

Reply:
Definitely. It is something to add to the next version